implants/golem/Cargo.toml

```diff
@@ -18,6 +18,7 @@ structopt = { workspace = true }
 itertools = { workspace = true }
 thiserror = { workspace = true }
 lsp-types = { workspace = true }
+walkdir = "2.5.0"
```

Comment:
walkdir was already added in the repo, im just adding it to this crate

Reply:
We try to use `{ workspace = true }` to import dependecies so we can define all deps at the workspace level.
This helps use the same dep version for everything for both stability and reducing build size.